search/internals.go

```diff
@@ -94,17 +94,17 @@ func setupFuncs(g graph.Graph, cost graph.CostFunc, heuristicCost graph.Heuristi
 
 /** Sorts a list of edges by weight, agnostic to repeated edges as well as direction **/
 
-type edgeSorter []concrete.WeightedEdge
+type byWeight []concrete.WeightedEdge
 
-func (e edgeSorter) Len() int {
+func (e byWeight) Len() int {
```

Comment:
minor nit: should the receiver value by named something else than `e`?
(`sort.{Int,Float,String}Slice` receivers are named `p`.)